object-store-client-play-26/src/main/scala/uk/gov/hmrc/objectstore/client/play/PlayObjectStoreContentReads.scala

```diff
@@ -21,52 +21,44 @@ import akka.stream.Materializer
 import akka.stream.scaladsl.Source
 import akka.util.ByteString
 import play.api.libs.json.{JsResult, JsValue, Json, Reads}
-import play.api.libs.ws.WSResponse
 import uk.gov.hmrc.objectstore.client.model.http.ObjectStoreContentRead
-import uk.gov.hmrc.objectstore.client.model.objectstore
 
 import scala.util.{Failure, Success, Try}
 
 trait PlayObjectStoreContentReads {
 
-  implicit def identity[F[_]]: ObjectStoreContentRead[F, Source[ByteString, NotUsed], Source[ByteString, NotUsed]] =
-    new ObjectStoreContentRead[F, Source[ByteString, NotUsed], Source[ByteString, NotUsed]] {
-      override def readContent(response: F[Option[objectstore.Object[Source[ByteString, NotUsed]]]]): F[Option[objectstore.Object[Source[ByteString, NotUsed]]]] = response
-    }
-
-  implicit def akkaSourceContentRead[F[_]](implicit F: PlayMonad[F]): ObjectStoreContentRead[F, WSResponse, Source[ByteString, NotUsed]] =
-    new ObjectStoreContentRead[F, WSResponse, Source[ByteString, NotUsed]] {
-      override def readContent(response: F[Option[objectstore.Object[WSResponse]]]): F[Option[objectstore.Object[Source[ByteString, NotUsed]]]] = {
-        F.map(response)(_.map(obj => obj.copy(content = obj.content.bodyAsSource.mapMaterializedValue(_ => NotUsed))))
-      }
+  implicit def akkaSourceContentRead[F[_]](implicit F: PlayMonad[F]): ObjectStoreContentRead[F, ResBody, Source[ByteString, NotUsed]] =
```

Comment:
Will a low priority implicit  `ObjectStoreContentRead[F, T, T]` will work here?

Reply:
It could, but we don't need one since all ContentReads start with ResBody - I think this is easier to see what is supported.